app/templates/gulpfile.js

```diff
@@ -170,6 +170,7 @@ gulp.task('serve:test', () => {
 <% if (includeBabel) { -%>
   gulp.watch('app/scripts/**/*.js', ['scripts']);
 <% } -%>
+  gulp.watch('test/index.html').on('change', reload);
   gulp.watch('test/spec/**/*.js').on('change', reload);
```

Comment:
`gulp.watch()` can accept an array, so just include it here. No need for multiple listeners.